core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -647,10 +647,10 @@ public enum CoreError implements ScalarDbError {
           + "If you want to modify a condition, please use clearConditions() to remove all existing conditions first",
       "",
       ""),
-  ENCRYPTED_COLUMNS_NOT_SUPPORTED(
+  TRANSPARENT_DATA_ENCRYPTION_NOT_ENABLED(
       Category.USER_ERROR,
       "0143",
-      "Encrypted columns are not supported in the ScalarDB Community edition",
+      "ScalarDB Transparent Data Encryption is not enabled. Enable ScalarDB Transparent Data Encryption to use this feature. Note that this feature is supported only in the ScalarDB Enterprise edition",
```

Comment:
The following suggestion is similar to [my suggestion for the ScalarDB Auth error message]().

```suggestion
      "ScalarDB Transparent Data Encryption is not enabled. To use ScalarDB Transparent Data Encryption, you must enable it. Note that this feature is supported only in the ScalarDB Enterprise edition",
```